src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -320,8 +321,14 @@ public void afterNavigateTo(String url, WebDriver driver) {
       } else {
         if (driver.getCurrentUrl().contains("data:text/html,chromewebdata ")) {
           driver.get(url);
+          logWarning("Url after navigation", driver.getCurrentUrl());
+        } else if (driver.getCurrentUrl().contains(URLsContent.NOT_A_VALID_COMMUNITY)) {
+          logWarning("Url after navigation", driver.getCurrentUrl());
+          driver.get("http://www.google.pl");
+          PageObjectLogging.log("Workaround", "Redirected to google page", true);
+//          throw new SkipException("Invalid url, redirected to Not_a_valid_community page");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)